pom.xml

```diff
@@ -19,40 +19,31 @@
         <artifactId>jberet-parent</artifactId>
         <version>2.1.1.Final</version>
     </parent>
-
     <artifactId>jberet-jpa-repository</artifactId>
+    <version>2.1.1.Final-SNAPSHOT</version>
 
     <properties>
-        <version.org.hibernate>5.6.2.Final</version.org.hibernate>
+        <version.jakarta.persistence>3.1.0</version.jakarta.persistence>
+        <version.org.hibernate>6.1.1.Final</version.org.hibernate>
+        <version.junit>4.13.2</version.junit>
+        <version.com.h2database>2.1.214</version.com.h2database>
```

Comment:
It's okay to use more recent version of h2 for this project. The parent pom has 1.4.197 and so is WildFly. So just note it may work differently.

Reply:
1.4.197 is also fine, just tested it right now